test/test_rpc.py

```diff
@@ -90,7 +92,13 @@ def wrapper(self):
     sys.version_info < (3, 0),
     "Pytorch distributed rpc package " "does not support python2",
 )
+@unittest.skipIf(C.is_asan, "Skip ASAN since torch + multiprocessing + fbcode doesn't work with asan")
 class RpcTest(MultiProcessTestCase):
+    @classmethod
+    def setUpClass(cls):
+        multiprocessing.set_start_method("spawn")
+        super(RpcTest, cls). setUpClass()
```

Comment:
Initialization of the super class is usually done first. Otherwise it could override whatever you're executing here.

Is there a reason for the reversed order here?